WooCommerce/Classes/ViewRelated/Orders/Order Details/Order Summary Section/SummaryTableViewCell.swift

```diff
@@ -7,6 +7,61 @@ struct SummaryTableViewCellPresentation {
     let statusName: String
 }
 
+/// The ViewModel for `SummaryTableViewCell`.
+///
+/// TODO This and that cell class should be renamed to be less ambiguous.
+///
+struct SummaryTableViewCellViewModel {
+    private let billingAddress: Address?
+    private let dateCreated: Date
+    private let orderNumber: String
+
+    let presentation: SummaryTableViewCellPresentation
```

Comment:
nit: this struct seems to be for displaying the order status UI, does it make sense to have a more descriptive name like `OrderStatusViewModel` nested under `SummaryTableViewCellViewModel`? I understand this struct is pre-existing, so up to you :)

Reply:
nit: could this property be `fileprivate`?

Reply:
I kind of like the `*Presentation` suffix because it was clear in how it was supposed to use. I think the property names were just not clear enough. Maybe this will be better:

1. 1e9f186d0 Rename `SummaryTableViewCellPresentation` to `OrderStatusPresentation`
2. 22d6f8b7e Clarify the `OrderStatusPresentation` property names
3. 1748cc4b6 Nest `OrderStatusPresentation` under the `CellViewModel`

https://github.com/woocommerce/woocommerce-ios/blob/1748cc4b6c51aac604af8524701fd30a36d81c3f/WooCommerce/Classes/ViewRelated/Orders/Order%20Details/Order%20Summary%20Section/SummaryTableViewCell.swift#L10-L13

What do you think? 🙂 

---

> nit: could this property be fileprivate?

Fixed in 3bec0a3cd29b03e61f10e0d4f4e986232fe2250c.

Reply:
yea `OrderStatusPresentation` looks way more clear! I wonder when we pick `*ViewModel` vs. `*Presentation` - like if there's no interaction handling, we use `*Presentation`?

Reply:
I'm not sure if the `*Presentation` should be a standard naming pattern. From our interactions throughout all the previous PRs, I think you're right that we should use the `*ViewModel` as much as possible. 🤔 What I know for sure right now is that we should document these patterns to see where we are at. 

<img src="https://user-images.githubusercontent.com/198826/82239296-cb185380-98f5-11ea-9086-fccd6848dbff.png" width="240">

I tend to forget a lot of things if they're all just buried in the code. 😅 We'll get to those Continuous Improvements tasks someday!